test/parallel_api/experimental/radix_sort_esimd.pass.cpp

```diff
@@ -370,6 +399,8 @@ int main()
             test_general_cases<uint16_t>(size);
             test_general_cases<int     >(size);
             test_general_cases<uint32_t>(size);
+            test_general_cases<uint64_t>(size);
+            test_general_cases<int64_t >(size);
```

Comment:
Could you remove TODO notes about support of these types from `esimd_radix_sort.hpp`?

Reply:
which one?

Reply:
Sorry, it seems we do not have such TODOs. We have `/ TODO: support double and small-size integers` which partially lost its relevance.